documentation/code-review/code-review.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: default
+title: Code Review Manual
+nav_order: 1
+---
+
```

Comment:
It would be helpful to add a brief intro to explain what a code review is, what should be accomplished, goals, etc. and also introduce what a pull request is.